packages/yoshi-flow-monorepo/src/verify-typescript-references.ts

```diff
@@ -16,26 +16,34 @@ export default async function verifyTypeScriptReferences({
         .map(([depName]) => {
           return path.relative(pkg.location, graph.get(depName)!.location);
         })
+        .sort()
         .map((relativePath) => ({ path: relativePath }));
 
+      const originalReferencesSorted = [...(tsconfig.references || [])]
+        .map(({ path }) => path)
+        .sort()
+        .map((path) => ({ path }));
```

Comment:
```suggestion
sort((a,b) => a.path > b.path ? 1 : -1)
```

obviously not critical